### PR TITLE
Linux zfs skip header lines

### PR DESCRIPTION
**Pull request by mac133k**
This is the follow-up to [PR#1197](https://github.com/performancecopilot/pcp/pull/1197)

### CODE REVIEW

File: src/pmdas/linux_zfs/zfs_dmu_tx.c [hunk 1]
```diff
@@ -39,10 +39,12 @@ zfs_dmu_tx_refresh(zfs_dmu_tx_t *dmu_tx)
             mname = strtok(line, delim);
             mval  = strtok(NULL, delim); // not used
             mval  = strtok(NULL, delim);
-            value = strtoull(mval, NULL, 0);
 
-            if (strncmp(mname, "dmu_tx_", 7) != 0)
```
Comment (natoscott) [inline src/pmdas/linux_zfs/zfs_dmu_tx.c:44]: I think we still need this line...?  What if the file format was (perhaps changed in the future) such that a different string prefix existed for some lines in the file?

File: src/pmdas/linux_zfs/zfs_dnodestats.c [hunk 2]
```diff
@@ -39,10 +39,12 @@ zfs_dnodestats_refresh(zfs_dnodestats_t *dnodestats)
             mname = strtok(line, delim);
             mval  = strtok(NULL, delim); // not used
             mval  = strtok(NULL, delim);
-            value = strtoull(mval, NULL, 0);
 
-            if (strncmp(mname, "dnode_", 6) != 0)
```
Comment (natoscott) [inline src/pmdas/linux_zfs/zfs_dnodestats.c:44]: Ditto.

File: src/pmdas/linux_zfs/zfs_zilstats.c [hunk 3]
```diff
@@ -39,10 +39,12 @@ zfs_zilstats_refresh(zfs_zilstats_t *zilstats)
             mname = strtok(line, delim);
             mval  = strtok(NULL, delim); // not used
             mval  = strtok(NULL, delim);
-	    value = strtoull(mval, NULL, 0);
 
-            if (strncmp(mname, "zil_" , 4) != 0)
```
Comment (natoscott) [inline src/pmdas/linux_zfs/zfs_zilstats.c:44]: Ditto.

**Comment by mac133k** [comment]
@natoscott Yes, you are right.

**Pull request merged**